mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -207,7 +227,13 @@ class WidgetAdapter(
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
         val wasStarted = holder.stop()
-        holder.vhc = ViewHolderContext(connection, bottomSheetPresenter, colorMapper, serverFlags, chartTheme)
+        holder.vhc = ViewHolderContext(
+            connection,
+            fragmentPresenter,
+            colorMapper,
+            serverFlags,
+            chartTheme
+        )
```

Comment:
Can revert the reformatting now, I guess?

Reply:
done